### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by pyrho**
🎉

### CODE REVIEW

File: embark-theme.conf [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Embark (https://github.com/embark-theme/kitty) {{{
+background #1e1c31
+foreground #EEFFFF
+cursor #A1EFD3
+selection_background  #fbfcfc
+
+# black
+color0 #1e1c31
+color8 #3E3859
```
Comment (skbolton) [inline embark-theme.conf]: This color has just recently changed. Can you update this to #585273?

**Review by skbolton — CHANGES_REQUESTED** [review]

File: embark-theme.conf (hunk 1 above)
Comment (pyrho) [reply embark-theme.conf]: Sure thing.

**Comment by skbolton** [comment]
I will preview this tonight on my machne just as a last minute check on things. But from what I see things look good.

File: embark-theme.conf [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# Embark (https://github.com/embark-theme/kitty) {{{
```
Comment (skbolton) [inline embark-theme.conf]: How would you feel about removing the fold markers? I would like to keep these files as clean as possible.

File: embark-theme.conf [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+active_border_color #A1EFD3
+inactive_border_color #a6b3cc
+bell_border_color #f56574
+
+active_tab_foreground   #222433 
+active_tab_background   #63f2f1
+active_tab_font_style   bold
+
+inactive_tab_foreground #63f2f1
+inactive_tab_background #222433
```
Comment (skbolton) [inline embark-theme.conf]: Small tweak I would like to ask for here just to keep this as close to the vim status line as possible. For the inactive tab bar styles what do you think about this:
```
inactive_tab_foreground #a6b3cc
inactive_tab_background #585273
```

Here is how it looks:
Screen Shot 2020-08-28 at 1 09 29 PM

What are your thoughts?

Comment (pyrho) [reply embark-theme.conf]: Sounds good, amending rn.

**Pull request merged**